source/sass/blocks/about-service.scss

```
@@ -0,0 +1,22 @@
+.about-service {
+  margin-top: -43px;
+  padding-top: 30px;
+  padding-left: 45px;
+  padding-bottom: 65px;
+  background: #141b3b url("../../img/about-mobile@1x.png") center top no-repeat;
```

```suggestion
  background: #141b3b url("../img/about-mobile@1x.png") center top no-repeat;
```
Лишние символы `../`.